tests/bootstrap.php

```diff
@@ -17,6 +17,6 @@ function dd() { call_user_func_array('dump', func_get_args()); die(); }
 ]);
 
 $loader = new Nette\Loaders\RobotLoader;
-$loader->setCacheStorage(ServiceLocator::getCacheStorage());
+$loader->setTempDirectory(__DIR__ . '/temp');
```

Comment:
Please refactor to `ServiceLocator::getTempDir()` and use it here and in `ServiceLocator::getCacheStorage()`.